partitura/musicanalysis/meter.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Meter numerator, Beat, and Tempo estimation.
+
+Implementation adapted from Jakob Woegerbauer 
+based on a model published by Simon Dixon.[1]
+
+References
+----------
+
+.. [1] Simon Dixon (2001), Automatic extraction of 
+    tempo and beat from expressive performances.
+    Journal of New Music Research, 30(1):39–58
+"""
+
+import warnings
+import numpy as np
+# import scipy.spatial.distance as distance
+# from scipy.interpolate import interp1d
+
+from partitura.utils import get_time_units_from_note_array, ensure_notearray, add_field
+
+
+#__all__ = ["estimate_tonaltension"]
+
+# Scaling factors
+MAX = 9999999999999
+MIN_INTERVAL = 0.01
+MAX_INTERVAL = 2  # in seconds
+CLUSTER_WIDTH = 1/12  # in seconds
+N_CLUSTERS = 100
+INIT_DURATION = 10  # in seconds
+TIMEOUT = 10  # in seconds
+TOLERANCE_POST = 0.4  # propotion of beat_interval
+TOLERANCE_PRE = 0.2  # proportion of beat_interval
+TOLERANCE_INNER = 1/12 
+CORRECTION_FACTOR = 1/4  # higher => more correction (speed changes)
+MAX_AGENTS = 100  # delete low-scoring agents when there are more than MAX_AGENTS
+CHORD_SPREAD_TIME = 1/12 # for onset aggregation
+
+
+class MultipleAgents():
```

Comment:
Documentation missing on MultipleAgents Class. Also this class has no Init call.

Reply:
I add documentation, the missing init call is no problem afaik.

Reply:
Sorry for the many comments, I just wanted to try the review process.